up42/processing.py

```diff
@@ -42,6 +42,17 @@ class JobMetadata(TypedDict):
     updated: str
 
 
+class JobSorting:
+    process_id = utils.SortingField("processID")
+    status = utils.SortingField("status")
+    created = utils.SortingField("created")
+    credits = utils.SortingField("creditConsumption.credits")
+
+
+def _to_datetime(value: Optional[str]):
```

Comment:
As we moved out this method from the `Job` class, and there is a method in `utils` named `format_time`. Should we move this to utils and group with the format_time method?

Reply:
Only if we see reuse